strings/src/main/res/values/strings.xml

```diff
@@ -1186,4 +1186,9 @@
     <string name="crash_app_summary">Force an uncaught exception causing the app to crash</string>
     <string name="predicate_caching">Predicate caching</string>
 
+    <string name="permission_dialog_title">About permissions</string>
+    <string name="permission_dialog_text">You will be asked to allow ODK Collect access to the features below, select “allow” if you want to use them.</string>
+    <string name="notifications">Notifications</string>
+    <string name="required_for_notifications">Required for notifications</string>
```

Comment:
This string is definitely not the final version. Glad to take any suggestions!

Reply:
Maybe it would be good to emphasize that those notifications are used to inform about the state/result of background jobs like auto-send and auto-download.